src/components/standalone/update/UploadImageDrawer.vue

```diff
@@ -32,17 +32,10 @@ const uploadedImageFileName = ref('')
 
 const fileToUpload = ref<File | null>(null)
 
-async function handleFileUpload(event: Event) {
+async function handleFileUpload() {
   fileInputValidationError.value = ''
   isReadyToUpdate.value = false
   isUploadingImage.value = false
```

Comment:
```suggestion
  isUploadingImage.value = false
  if (!fileToUpload.value) return
```
A check on the uploaded file should probably stll be kept, since there are cases where the file can be empty (for example, when the user cancels the file selection window). I don't think it makes much sense validating the file when it's not even selected